react-app/src/stylesheets/Page.scss

```diff
@@ -126,6 +126,10 @@
   margin-top: 0px;
 }
 
+.button-text {
+  margin-bottom: 0px;
+}
+
```

Comment:
I think we can remove this now since we've gone with the LinkButton component right?

Reply:
I've removed this.